### ISSUE TITLE
parseErrorPretty without position

### ISSUE THREAD

**Issue by user471** [mrkkrp/megaparsec#137]
I want to get only a error text from `ParseError`. So I need a function like `sourcePosStackPretty` but only with error text.
Something like `parseErrorTextPretty `

```
parseErrorPretty (ParseError pos us ps xs) =
  sourcePosStackPretty pos ++ ":\n" ++
  if E.null us && E.null ps && E.null xs
    then "unknown parse error\n"
    else concat
      [ messageItemsPretty "unexpected " us
      , messageItemsPretty "expecting "  ps
      , unlines (showErrorComponent <$> E.toAscList xs) ]
```
=>
```
parseErrorPretty e@(ParseError pos us ps xs) =
  sourcePosStackPretty pos ++ ":\n" ++ parseErrorTextPretty e

parseErrorTextPretty (ParseError pos us ps xs) =
  if E.null us && E.null ps && E.null xs
    then "unknown parse error\n"
    else concat
      [ messageItemsPretty "unexpected " us
      , messageItemsPretty "expecting "  ps
      , unlines (showErrorComponent <$> E.toAscList xs) ]

```

**Comment by mrkkrp** [comment]
It looks like you already have one. What is the actual question? Do you the `parseErrorTextPretty` added to the library itself? I don't think that displaying just text without position is common enough need. In fact the position info is probably the most important information in the entire erorr message.

**Comment by user471** [comment]
> Do you want the parseErrorTextPretty added to the library itself? 

Yes, it's a suggestion for refactoring.
> I don't think that displaying just text without position is common enough need. 

For any language compiler it should be pretty common. A compiler has many different errors. All errors have error text and position. But right now It's not possible to separate error and position from parsing error. 

Another example - convert parsing errors to Json format `[{row : 1, column : 1, errorText : "foo" }]`

**Comment by mrkkrp** [comment]
The function will be available in version 5.1.0.
